util/cmd/release/main.go

```diff
@@ -109,9 +109,98 @@ func main() {
 	util.Execute(
 		"ghr",
 		"-t="+token,
+		"-n="+version,
+		"-b='"+notes(version)+"'",
 		"-u=googleapis",
 		"-r=gapic-generator-go",
 		"-c="+commitish,
 		version,
 		stagingDir)
 }
+
+// notes collects the commit messages since the previous tag, aggregates
+// dependency updates into one message, and returns formatted release notes.
+func notes(version string) string {
+	previous := util.ExecuteWithOutput(
+		"git",
+		"describe",
+		"--abbrev=0",
+		"--tags",
```

Comment:
Does this need `--no-pager`?

Reply:
Nope, this returns a single tag, just the tag preceding `version`.

For example, in this repo:
```
$ git describe --abbrev=0 --tags v0.12.5^
v0.12.4
```

Number of `^` used indicates how many to jump back to.
```
$ git describe --abbrev=0 --tags v0.12.5^^
v0.12.3
```

At least, that is what I've seen in local testing.